Anlab.Mvc/Services/DbInitializationService.cs

```diff
@@ -49,6 +49,33 @@ public DbInitializationService(ApplicationDbContext context, UserManager<User> u
                 "AL-KCL",
                 "AL-OX",
                 "AL-PY",
+                "-BCL-P-IC",
```

Comment:
@srkirkland Lets me see what comes back. For real data, we wouldn't include any of those in the DB so they would just get filtered out.